src/Popover/Popover.tsx

```diff
@@ -165,18 +235,18 @@ export default class Popover extends React.Component {
     };
   };
 
-  getPopoverNode = () => {
-    let popover;
+  getPopoverNode = (): Element => {
+    let popover: Element;
     if (this.isAbsolute()) {
-      popover = this.popoverNode;
+      popover = this.popoverNode!; // FIXME(gabro): is this safe?
```

Comment:
this doesn't change the current behavior, so it's probably safe, still I would like to double check to see whether we have potential bugs.